nnpdf_data/nnpdf_data/new_commondata/NNPDF_POS_2P24GEV/metadata.yaml

```diff
@@ -56,7 +56,7 @@ implemented_observables:
     description: Deep Inelastic Scattering
     label: 'positivity dataset: DIS $c+\bar{c}$ structure function $F_2^c$'
     units: ''
-  process_type: DIS_F2C
+  process_type: POS_F2C
```

Comment:
Why this change? The process is DIS_F2C, the only difference is that we use the dataset in a lagrange multiplier. Also, this change is not mirrored in `process_options.py`

Reply:
I think it is better to keep it as a `POS_F2C`. The positivity are special. I would however call them all `POS_F2`?
The other reason why I think they should not be called `DIS_` is that there's a lot of hidden logic that treats `DIS_` processes differently, better  to keep the positivity ones differentiated.

Agree that a `POS_F2` would be needed in process type (I guess you are already doing it like that but positivity/integrability datasets don't need kinematic coverage so no need to implement one).

Reply:
Just to be sure, would you change all process types starting with `POS_F2` with `POS_F2` (eg. `POS_F2C_CCP ` -> `POS_F2`)

What about other processes such as `POS_FLL` should we just leave them as they are?

Reply:
I did rename all of the `POS_F2*` process types to `POS_F2`. I have also added the corresponding process types in `process_options.py`